lib/railties/lib/rails/templates/components/_preview.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="preview" id="<%= example %>">
```

Comment:
Per my read of the Rails source, wouldn't this file go in `railties/lib/rails/templates/rails/components`?

Reply:
yes you're right. I missed the second `rails` folder.

Btw, is it ok that we define these files here in AC:V lib folder? 

I'm aware that these kind of templates actually live in the railties gem, but I wasn't sure where to locate them.

Reply:
Hmm. That I don't know. I think as long as we get things _close_, we'll be fine. We're going to have to make other tweaks when merging this upstream, so I'm not worried about getting this perfect 😄